deno/payloads/v8/user.ts

```diff
@@ -148,7 +148,9 @@ export enum UserFlags {
 	 */
 	Spammer = 1 << 20,
 	/**
-	 * @unstable User's account has been quarantined based on recent activity
+	 * User's account has been quarantined based on recent activity
+	 *
+	 * @unstable This user flag is currently not documented by Discord but has known values which we will try to keep up to date.
```

Comment:
Can you add this to the spammer flag too

Reply:
I was about to do that, but I don't know if it would be better if I open another PR with that.

Reply:
That's fine with me!

Reply:
Done! #500